webapp/src/test/java/io/github/microcks/util/asyncapi/AsyncAPIImporterTest.java

```diff
@@ -37,904 +37,940 @@
 
 /**
  * This is a test case for class AsyncAPIImporter.
+ *
  * @author laurent
  */
 public class AsyncAPIImporterTest {
 
-   @Test
-   public void testSimpleAsyncAPIImportYAML() {
-      AsyncAPIImporter importer = null;
-      try {
-         importer = new AsyncAPIImporter("target/test-classes/io/github/microcks/util/asyncapi/user-signedup-asyncapi.yaml", null);
-      } catch (IOException ioe) {
-         ioe.printStackTrace();
-         fail("Exception should not be thrown");
-      }
-
-      importAndAssertOnSimpleAsyncAPI(importer);
-   }
-
-   @Test
-   public void testSimpleAsyncAPIImportJSON() {
-      AsyncAPIImporter importer = null;
-      try {
-         importer = new AsyncAPIImporter("target/test-classes/io/github/microcks/util/asyncapi/user-signedup-asyncapi.json", null);
-      } catch (IOException ioe) {
-         ioe.printStackTrace();
-         fail("Exception should not be thrown");
-      }
+  @Test
+  public void testSimpleAsyncAPIImportYAML() {
```

Comment:
File was indented with 2 spaces instead of 3. Can you please fix this to avoid noise?